lib/storage/file.js

```diff
@@ -54,9 +54,8 @@ var STORAGE_UPLOAD_BASE_URL = 'https://www.googleapis.com/upload/storage/v1/b';
  * @param {module:storage/bucket} bucket - The Bucket instance this file is
  *     attached to.
  * @param {string} name - The name of the remote file.
- * @param {object=} metadata - Metadata to set on the object. This is useful
```

Comment:
It wasn't actually used anywhere or even a thing. Back when we were working out the details of how creating a File object would work, we considered `new File()` having network side-effects. The operation was ultimately shut down, and I believe the `metadata` argument was briefly re-purposed as a way to set metadata from `bucket.getFiles()` when the API responds with the metadata. I decided it was most clear to just use `file.metadata = {/*...*/}`, and not even make it look like we accept a "metadata" object to a "new File()" call, because what would that imply? Would the metadata get set immediately if specified? That's wacky. Being that it's a privately documented property, the artifact of yesteryear was forgotten.